fuse/plugins/micro_physics/lineage_cluster.py

```diff
@@ -281,6 +300,24 @@ def get_parent(event_interactions, event_lineage, particle):
     return possible_parents[-1], possible_parents_lineages[-1]
 
 
+def get_all_particle_secondaries(event_interactions, particle):
+    """Returns all secondaries of the given particle.
+
+    It iterates over all interactions of the event and checks if the
+    parent id of the interaction is the same as the track id of the
+    particle, and continues down the tree until no more secondaries are
+    found or a RadioactiveDecayBase type interaction is found.
```

Comment:
The docstring reads like the loop is supposed to check for RadioactiveDecayBase type interactions and then stop. There is no such think in the loop and I was wondering if the code needs to be updated or the docstring.

Reply:
Thanks Henning, I had it like that at the beginning but then realised I need only first generation secondaries. Updating docstring now :)